models/selectArticles.js

```diff
@@ -0,0 +1,8 @@
+const db = require("../db/connection");
+
+exports.selectArticles = ({ article_id: id }) => {
```

Comment:
Ideally it would be preferable of this destructuring of the id was done in the controller. That is the controllers job to prepare data for the model 👍